.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!-- Follow this template for PR's -->
+
+<!--Example Template for adding feature of binary search function
+ -->
+
+<!-- Short description of your changes -->
+
+Adding code snippet of binary search function in algorithms directory
+
+<!-- Add issue no. which corresponds to this PR -->
```

Comment:
*if applicable*; most PRs will be adding algorithms and we don't allow issues for missing algorithms